modules/understanding-upgrade-channels.adoc

```diff
@@ -82,6 +83,27 @@ and pre-production and production environments that participate in connected cus
 You can use the `stable-{product-version}` channel to upgrade from a previous minor version of
 {product-title}.
 
+[discrete]
+== eus-{product-version} channel
+
+In addition to the stable channel, certain minor versions of {product-title} offer an
+Extended Update Support[https://access.redhat.com/support/policy/updates/openshift#ocp4_phases] (EUS).
+These EUS versions extend the maintenance phase for customers with Premium Subscriptions
+to 14 months. {product-title} {product-version} is a minor version that includes EUS.
+
+Although there is no difference between stable-{product-version} and eus-{product-version}
+channels until {product-version} transitions to EUS phase, you can switch
+to the EUS channel as soon as it becomes available. When {product-title} {product-version}
+transitions to the EUS phase of its lifecycle, the stable-{product-version} channel will
+no longer receive updates. After you upgrade a to a version that is exclusive
+to the EUS channel, that cluster is no longer be eligible for minor version upgrades
```

Comment:
"is no longer be eligible" -> "will no longer be eligible" or "is no longer eligible"